deploy/helm/templates/beyla/configmap.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if .Values.beyla.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.fullname" (tuple . "-beyla-config") }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "common.labels" . | indent 4 }}
+    app.kubernetes.io/component: config
+  annotations:
+{{ include "common.annotations" . | indent 4 }}
+data:
+  beyla-config.yml: |
+    otel_metrics_export:
+      endpoint: http://{{ include "common.fullname" (tuple . "-metrics-collector") }}:{{ .Values.otel.metrics.otlp_endpoint.port }}
+      protocol: grpc
+      interval: {{ quote .Values.otel.metrics.prometheus.scrape_interval }}
+      features: ["application", "application_process", "application_service_graph", "network"]
+    ebpf:
+      enable_context_propagation: true
+    process:
+      enabled: true
+    network:
+      enable: true
+    discovery:
+      services:
+        - k8s_namespace: .*
+      exclude_services:
+        - exe_path: ".*otelcol.*|.*beyla.*"
+    log_level: {{ .Values.beyla.telemetry.logs.level }}
+    attributes:
+      kubernetes:
+        enable: true
+    prometheus_export:
+      port: 9090
+      path: /metrics
+    internal_metrics:
+      prometheus:
+        port: 6060
+        path: /metrics
```

Comment:
Are these temporary for alphas. Or do we want to keep them also once released?

Reply:
I would remove them later, I would like to see if there is anything useful there